src/server/middleware/createRequestHtmlFragment.jsx

```diff
@@ -17,11 +17,16 @@
 import React from 'react';
 import { Provider } from 'react-redux';
 import url, { Url } from 'url';
+import util from 'util';
 import { RouterContext, matchPromise } from '@americanexpress/one-app-router';
 import { composeModules } from 'holocron';
 import createCircuitBreaker from '../utils/createCircuitBreaker';
 
-import { getRenderMethodName, renderForStaticMarkup, renderForString } from '../utils/reactRendering';
+import {
```

Comment:
are these the formatting changes you were talking about when using a `vscode` editorconfig?

Reply:
I think this is coming from having `prettier` enabled in my VSCode